src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1636,7 +1636,8 @@ private StanzaError onSessionAcceptInternal(
             logger.info(participant + " did not advertise any SSRCs. Injecting " + ssrc);
             sourcesAdvertised
                     = new EndpointSourceSet(
-                            new Source(ssrc, MediaType.AUDIO, null, true));
+                            // TODO figure out what source name should be in this case
```

Comment:
For reference: the injected sources are not signaled to clients, and will be removed with the secure octo changes we're working on right now